examples/failing/ConflictingExports2.purs

```diff
@@ -1,13 +0,0 @@
--- @shouldFailWith ScopeConflict
-module A where
-
-  thing :: Int
-  thing = 1
-
--- Fails here because re-exporting forces any scope conflicts to be resolved
-module Main (thing, module A) where
-
-  import A
-
-  thing :: Int
-  thing = 2
```

Comment:
This is now a warning rather than error, so the test no longer fails (`thing` is resolved to `Main.thing` due to it being a local declaration, and therefore always preferred over a conflicting value from an open import).